python/py-datasets/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-datasets
+version             1.1.2
+platforms           darwin
```

Comment:
it this port doesn't compile anything (please check!), then add `supported_archs noarch`

Reply:
Thanks done.